rapidast.py

```diff
@@ -202,15 +212,20 @@ def run():
 
     # Run all scanners
     scan_error_count = 0
-    for name in config.get("scanners"):
-        logging.info(f"Next scanner: '{name}'")
-
-        ret = run_scanner(name, config, args, scan_exporter)
-        if ret == 1:
-            logging.info(f"scanner: '{name}' failed")
-            scan_error_count = scan_error_count + 1
-        else:
-            logging.info(f"scanner: '{name}' completed successfully")
+    try:
```

Comment:
Added a try/catch block to handle any unhandled exceptions, ensuring that the configuration dump is done